docs/data/stashcache/install-origin.md

```diff
@@ -1,6 +1,12 @@
 Installing the StashCache Origin
 ================================
 
+!!! warning
+    If you want to run origins for authenticated and unauthenticated data,
+    you **must** run them on separate hosts.
```

Comment:
This is not true you just need separate mounts and separate ports.

Reply:
I don't believe that's the case because of the config of the shared `cmsd`: https://opensciencegrid.atlassian.net/browse/SOFTWARE-3559

Reply:
Then the solution is not to share the same cmsd. I mean I am ok with separate containers for origin-auth and non-auth
Seems the clean alternatives

Reply:
Oh man, we should've thought of that earlier! That's a much cleaner solution than sharing the `cmsd` and adding a cmsd Topology endpoint for generating its authfile. Do you see any issues with this idea @matyasselmeci ?

I think for the time being, this particularly documentation is correct until we resolve this at the packaging level.

Reply:
They don't share the same cmsd. There's a cmsd@stash-origin and a cmsd@stash-origin-auth.
I think you could make it work if you used a different export for the two instances.

Reply:
I'm going to go ahead and resolve this and move ahead with the merge since we still need to address https://opensciencegrid.atlassian.net/browse/SOFTWARE-3559 before having a single origin host for auth + unauth data